InstanceEditor.qml

```diff
@@ -0,0 +1,100 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.5
+
+// Component to edit/fixup FHIR instances as easily as possible
+ScrollView {
+    property string myText
+    property string fontName
+
+    /** Scroll the text to the specified line number
+      * @param lineNumber the line number, with counting started at 1
+      */
+    function scrollToLine(lineNumber) {
+        // We can't calculate the y position directly because a logical and
+        // displayed line might differ in height due to line wrapping. QML's
+        // positionToRectangle() function can tell us what we need, but for
+        // that we have to calculate the position in the text string of the
+        // line.
+        var position = 0
+        var lines = textArea.text.split("\n")
+        for (var currentLine = 0; currentLine < lineNumber - 1; currentLine++) {
+            position += lines[currentLine].length + 1 // include newline char
+        }
+        contentItem.contentY = textArea.positionToRectangle(position).y
+    }
+
+    property int selectStart
+    property int selectEnd
+    property int curPos
+
```

Comment:
These don't have to be 'public' properties, you can just make them local variables in the openContextMenu() function below.

Reply:
Applied feedback.